pom.xml

```diff
@@ -313,6 +313,24 @@
                 </plugins>
             </build>
         </profile>
+        <!-- add JAXB to module path in Java 9 -->
+        <profile>
+            <id>java9</id>
+            <activation>
+                <jdk>9</jdk>
+            </activation>
+            <properties>
+                <jaxb.version>2.2.2</jaxb.version>
+            </properties>
+            <dependencies>
+                <dependency>
+                    <groupId>javax.xml.bind</groupId>
+                    <artifactId>jaxb-api</artifactId>
+                    <version>${jaxb.version}</version>
+                    <scope>provided</scope>
```

Comment:
it's not provided at runtime by default. So executing it with Java 9+ will require users to explicitly add it again, right? Did you try that?

Reply:
Did not try for Java version != 9

Reply:
yes, it works with Java 9